src/interactions/interactionLoader.ts

```diff
@@ -29,7 +30,9 @@ export async function loadCommands(): Promise<Collection<string, Command>> {
 			.filter(isActive)
 			.map(async (filename) => {
 				const _command = (await import(`../commands/${filename}`)).default as Command;
-				loadedCommands.set(_command.name, _command);
+				if (config.logActivity || _command.name !== "game-activity-tracker") {
```

Comment:
If logActivity is true every command should be loaded and if it's not every command except the activity-tracker should be loaded. I thought it would make sense that the whole command is just disabled when no activity is tracker. Without the logs the commands aren't useful.